packages/ckeditor5-fullscreen/src/handlers/abstracteditor.ts

```diff
@@ -122,4 +140,41 @@ export default class AbstractEditorHandler {
 			this._container = null;
 		}
 	}
+
+	/**
+	 * Stores the current state of the annotations UIs to restore it when leaving fullscreen mode.
+	 */
+	private _overrideAnnotationsUIs() {
+		const annotationsUIs = this._editor.plugins.get( 'AnnotationsUIs' ) as AnnotationsUIs;
+
+		this.annotationsUIsData = new Map( annotationsUIs.uisData );
+
+		// Switch to the wide sidebar.
+		const sidebarPlugin = this._editor.plugins.get( 'Sidebar' ) as Sidebar;
+
+		if ( !sidebarPlugin.container ) {
+			sidebarPlugin.setContainer(
+				this.getContainer().querySelector( '[data-ck-fullscreen="right-sidebar"]' ) as HTMLElement
+			);
+		}
+
+		annotationsUIs.switchTo( 'wideSidebar' );
```

Comment:
I'm not sure about this. It will display all annotations, ignoring any filters. Imagine that someone has a filter that hides very important annotations that should not be visible to others.

My proposal:

1. We check if someone has a filter defined for `wideSidebar`. If so, we retrieve and apply it in fullscreen.
2. If no filter is defined for `wideSidebar`, we read the filters for the active display mode and apply them.
3. If no filters are defined for the active display mode, we proceed without applying a filter.